src/components/Calculator.js

```diff
@@ -9,35 +10,70 @@ const Calculator = () => {
     next: null,
     operation: null,
   });
+  const [quotes, setQuotes] = useState([]);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      try {
+        const options = {
+          method: 'GET',
+          headers: { 'x-api-key': 'btAEiRrit9u1L1sZDM0Oew==si2rJMo7DAvCioHN' },
+        };
+        const res = await fetch(
+          'https://api.api-ninjas.com/v1/quotes?category=happiness',
+          options,
+        );
+        const json = await res.json();
+        setQuotes(json);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setQuotes, setIsLoading]);
+
+  if (hasError) return <div>Something went wrong!</div>;
+
+  if (isLoading) return <div>Loading...</div>;
```

Comment:
- [ ] Kindly move the logic for fetching quote data in the new component (Quotes.js) as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md#:~:text=Fetch%20data%20inside%20of%20a%20new%20component).